rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -228,17 +228,17 @@ impl SerialSubmitter {
 
         match self.process_message(&msg).await {
             Ok(true) => {
-                info!(id=?msg.message.id(), nonce=msg.message.nonce, "Message processed");
+                info!(message = %msg.message, "Message processed");
                 self.record_message_process_success(&msg)?;
                 return Ok(());
             }
             Ok(false) => {
-                info!(id=?msg.message.id(), nonce=msg.message.nonce, "Message not processed");
+                info!(message = %msg.message, "Message not processed");
             }
             // We expect this branch to be hit when there is unexpected behavior -
             // defined behavior like gas estimation failing will not hit this branch.
-            Err(err) => {
-                warn!(id=?msg.message.id(), nonce=msg.message.nonce, error=?err, "Error occurred when attempting to process message");
+            Err(error) => {
+                warn!(message = %msg.message, ?error, "Error occurred when attempting to process message");
```

Comment:
Hmm, should this be bumped `error!`? I'm not sure what might cause this to happen, woudl have to dig

Reply:
Yeah, might make sense. Part of me thinks that warn is good because failing to submit a message does not indicate an issue with the relayer, but on the other hand that would be `Ok(false)` this indicates something else happened. Changed.